qiskit/visualization/latex.py

```diff
@@ -612,18 +607,29 @@ def _add_condition(self, op, wire_list, col):
             cwire += 1
 
         gap = cwire - max(wire_list)
+        meas_offset = -0.3 if isinstance(op, Measure) else 0.0
         if self.cregbundle:
-            # Print the condition value at the bottom
-            self._latex[cwire][col] = "\\dstick{_{_{=%s}}} \\cw \\cwx[-%s]" % (
-                str(op.condition[1]),
+            # Print the condition value at the bottom and put bullet on creg line
+            self._latex[cwire][col] = "\\control \\cw^(%s){^{%s}} \\cwx[-%s]" % (
+                meas_offset,
+                str(hex(op.condition[1])),
                 str(gap),
             )
         else:
             # Add the open and closed buttons to indicate the condition value
-            for i in range(creg_size):
+            for i in range(creg_size - 1):
                 control = "\\control" if if_value[i] == "1" else "\\controlo"
                 self._latex[cwire + i][col] = f"{control} \\cw \\cwx[-" + str(gap) + "]"
                 gap = 1
+            # Add (hex condition value) below the last cwire
+            control = "\\control" if if_value[creg_size - 1] == "1" else "\\controlo"
+            self._latex[creg_size + cwire - 1][col] = (
+                f"{control}" + " \\cw^(%s){^{%s}} \\cwx[-%s]"
```

Comment:
The fix looks great! I was just curious to know if we could make a small aesthetic fix. In line 627, when we print the hex, it actually gets printed as a string inside a math environment. So it looks like 
![image](https://user-images.githubusercontent.com/51048173/131858483-ff4e26ec-2181-44b7-9d30-eeffde330f4d.png)
If we could just change this line to
```
f"{control}" + " \\cw^(%s){^{\\mathtt{%s}}} \\cwx[-%s]"
```
then we can get something like
![image](https://user-images.githubusercontent.com/51048173/131858817-17b8e982-2333-4577-9e7a-180692577785.png)
which I personally think looks more like a hexadecimal. Any thoughts on this?

Reply:
Agreed. That x was a bit too fancy, and the new form is more compact as well. Will make the change.